modules/core/src/test/java/org/apache/ignite/cache/affinity/AffinityHistoryCleanupTest.java

```diff
@@ -224,6 +228,54 @@ private void checkHistory(Ignite ignite, List<AffinityTopologyVersion> expHist,
         assert cnt > 4;
     }
 
+    /**
+     * @throws Exception If failed.
+     */
+    @Test
+    @WithSystemProperty(key = IGNITE_AFFINITY_HISTORY_SIZE, value = "10")
```

Comment:
Please, consider decreasing this number in order to reduce the number of PMEs. It can be 4 or 5, for instance.

Reply:
Thanks. I decreased to 1.